src/content/docs/dropins/user-auth/containers/sign-up.mdx

```diff
@@ -0,0 +1,85 @@
+</Diagram>
+
+## SignUp configurations
+
+The SignUp container provides the following configuration options:
+
+<OptionsTable
+    compact
+    options={[
+        ['Options', 'Type', 'Req?', 'Description'],
+        ['addressesData', 'AddressFormProps[]', 'No', 'Allows passing address data into the SignUp drop-in. On successful sign-up, corresponding addresses will be automatically created in the newly created account.'],
+        ['inputsDefaultValueSet', 'inputsDefaultValueSetProps[]', 'No', ' Allows passing default values for inputs (prefilled sign-up form).'],
+        ['fieldsConfigForApiVersion1', 'any', 'No', 'Allows passing a set of field configurations for environments that do not support the createCustomerV2 mutation (fallback, should not be used in environments with the latest AC version).'],
+        ['apiVersion2', 'boolean', 'No', 'Defaults to true. Allows switching to the createCustomer mutation if createCustomerV2 is not supported in the current version of AC (fallback, should not be used in environments with the latest AC version).'],
+        ['displayTermsOfUseCheckbox', 'boolean', 'No', 'Controls displaying of the appropriate checkboxes in the sign-up form. Note: Rendered checkboxes are not functional and should not be used yet. They will be used in the future.'],
+        ['displayNewsletterCheckbox', 'boolean', 'No', ''],
+        ['isAutoSignInEnabled', 'boolean', 'No', 'Determines if the user should be automatically signed in after registration. This has no effect if email confirmation is enabled in the admin panel (automatic sign-in is not possible in this case).'],
+        ['formSize', 'default | small', 'No', 'Controls form paddings and spacing. Use "small" to embed the form in small layout containers like a dropdown in the header.'],
+        ['hideCloseBtnOnEmailConfirmation', 'boolean', 'No', 'Controls the visibility of the “Close” button on the email confirmation view.'],
+        ['routeRedirectOnEmailConfirmationClose', 'function', 'No', 'Determines where the user is redirected when the “Close” button on the email confirmation view is clicked.'],
+        ['slots', 'function', 'No', 'Allows passing the SuccessNotification container or custom component rendered on successful sign-up if routeRedirectOnSignIn is not provided.'],
+        ['routeSignIn', 'function', 'No', 'Determines where the “Already a member? Sign in” link redirects the customer.'],
+        ['routeRedirectOnSignIn', 'function', 'No', 'Determines the page to which the user should be redirected after sign-in (if isAutoSignInEnabled is true). If provided, the user won’t see SuccessNotification or any override provided via slot.'],
+        ['onErrorCallback', 'function', 'No', ' Callback executed when an error occurs, receiving the error object as a parameter.'],
+        ['onSuccessCallback', 'function', 'No', 'Callback executed when the user successfully signs up, receiving userName and status as parameters.'],
```

Comment:
Additionally, we have introduced a new prop responsible for the "Confirm Password" field.

`requireRetypePassword`

PR with screenshot - https://git.corp.adobe.com/Commerce-Storefront/storefront-auth/pull/58